include/picongpu/fields/incidentField/profiles/FromOpenPMDPulse.hpp

```diff
@@ -79,6 +81,7 @@ namespace picongpu
                     {
                         //! User SI parameters
                         using Params = T_Params;
+                        using dataType = typename Params::dataType; // field record data type
```

Comment:
Please move the comment above the line, otherwise these comments cause formatting issues later on
```suggestion
                        // field record data type
                        using dataType = typename Params::dataType;
```

Reply:
Fixed.